storage/mounts.go

```diff
@@ -94,8 +94,8 @@ func (mp MountPoint) Mount(mo Mount, client *common.DatabricksClient) (source st
 				raise e
 		mount_source = safe_mount("/mnt/%s", "%v", %s, "%s")
 		dbutils.notebook.exit(mount_source)
-	`, mp.name, mo.Source(), extraConfigs, mp.encryptionType) // lgtm[go/unsafe-quoting]
-	result := mp.exec.Execute(mp.clusterID, "python", command)
+	`, mp.Name, mo.Source(), extraConfigs, mp.EncryptionType) // lgtm[go/unsafe-quoting]
```

Comment:
it's interesting that this directive doesn't help

Reply:
where did you see warning triggered? bot didn't comment and security tab is clear 🤷‍♂️

Reply:
https://github.com/databrickslabs/terraform-provider-databricks/pull/914/checks?check_run_id=4199083573